### PR TITLE
Slight improvements to Black-Red theme and custom icon

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
This is still not 100% - so there'll be a PR for that later, but hey there's a custom icon to go with the theme

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.kt [hunk 1]
```diff
@@ -56,8 +57,11 @@ import kotlinx.android.synthetic.main.main_activity.tabs
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.flow.launchIn
 import timber.log.Timber
+import android.content.pm.PackageManager
+import android.content.ComponentName
+import android.content.res.Configuration
 
-class MainActivity : BaseActivity<MainActivityBinding>() {
+open class MainActivity : BaseActivity<MainActivityBinding>() {
```
Comment (jobobby04) [inline app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.kt:64]: Is this needed?

Comment (jobobby04) [reply app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.kt:64]: I see now, there is a red activity at the bottom

Comment (AbdullahM0hamed) [reply app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.kt:64]: Red is an illusion

**Comment by AbdullahM0hamed** [comment]
soon™ - now I just gotta do minor theme stuff

**Comment by nate-moo** [comment]
> Look at me `https://github.com/jobobby04/TachiyomiSY/pull/97`
> Making a million commits

**Pull request closed without merging**